### PR TITLE
updated cli commands output stream as stdout

### PR DESCRIPTION
**Pull request by cc-shekher**

### CODE REVIEW

File: src/cli/cmd/version/version.go [hunk 1]
```diff
@@ -24,7 +24,7 @@ func NewCmd() *cobra.Command {
 }
 
 func cmdRunE(cmd *cobra.Command, args []string) (err error) {
-	cmd.Print(format(build.Version, build.Date))
+	fmt.Fprintf(cmd.OutOrStdout(), format(build.Version, build.Date))
```
Comment (kanaksinghal) [inline src/cli/cmd/version/version.go:27]: @cc-shekher I might have done that sorry! But maybe I'm miss-understanding something, I thought `cmd.Print` was short-hand for `fmt.Fprintf(cmd.OutOrStdout(),`. not true?

Comment (cc-shekher) [reply src/cli/cmd/version/version.go:27]: @kanaksinghal If you see the definition of `cmd.Print` it is as below:

`func (c *Command) Print(i ...interface{}) {
	fmt.Fprint(c.OutOrStderr(), i...)
}`

using `cmd.OutOrStdout()` will explicitly override that.

**Review by kanaksinghal — DISMISSED** [review]
you are right. I think I have used this in various other commands as well, which we'll need to switch back now.

**Comment by kanaksinghal** [comment]
before we merge, can you add a ticket number on this

**Review by kanaksinghal — APPROVED** [review]

**Pull request merged**